src/MaxwellSolvers/AbsorbingBC.h

```diff
@@ -0,0 +1,427 @@
+#ifndef ABC_H
+#define ABC_H
+#include <Kokkos_Core.hpp>
+#include "Types/Vector.h"
+#include "Field/Field.h"
+template<unsigned a, unsigned b>
+constexpr KOKKOS_INLINE_FUNCTION auto first(){
+    return a;
+}
+template<unsigned a, unsigned b>
+constexpr KOKKOS_INLINE_FUNCTION auto second(){
+    return b;
+}
+template<typename _scalar, unsigned _main_axis, unsigned... _side_axes>
+struct second_order_abc_face{
```

Comment:
I think it would be nice to have some more comments around the code e.g. in this struct what side_axis<n>_onehot is, and just in general what is going on throughout the computation.